.env.develop

```diff
@@ -44,7 +44,7 @@ REACT_APP_POLYGON_NODE_URL=https://dev-daemon.polygon.shapeshift.com
 REACT_APP_GNOSIS_NODE_URL=https://dev-daemon.gnosis.shapeshift.com
 REACT_APP_ARBITRUM_NODE_URL=https://dev-daemon.arbitrum.shapeshift.com
 REACT_APP_COSMOS_NODE_URL=https://dev-daemon.cosmos.shapeshift.com
-REACT_APP_THORCHAIN_NODE_URL=https://dev-daemon.thorchain.shapeshift.com/lcd
```

Comment:
We probably want to keep `/lcd` as a part of the env var so we don't have to plumb it through everywhere on change.

Reply:
See this PR, which intentionally did this: https://github.com/shapeshift/web/pull/5485

Reply:
For whatever reason, the csp rules weren't behaving well with the path so I punted and switched back. On the plus side, we have access to` /rpc` if we ever need without having to add a new env var I suppose..

Reply:
as discussed if this is causing csp issues let's leave as is